src/components/DonorDashboard/DonorDashboard.jsx

```diff
@@ -25,19 +133,27 @@ const DonorDashboard = ({ donationId }) => {
   }, [donationId]);
 
   return (
-    <Box bg="#edf1f9" minHeight="100vh">
+    <Box bg="#edf1f9" maxH="100%">
       <Grid templateColumns="repeat(3, 1fr)" gap={10} p="20px 40px 40px 40px">
         <GridItem colSpan={2}>
           <Text fontSize="30px" fontWeight="700" mb="20px">
-            My Forms
+            My Donation
           </Text>
-          <Box bg="white" w="100%" h="500" p={4} />
+          {displayBanner()}
+          <Box borderRadius="6px" bg="white" w="100%" h="500" p={4}>
+            <DonationDetails data={donation} />
+          </Box>
         </GridItem>
         <GridItem colSpan={1}>
-          <Text fontSize="30px" fontWeight="700" mb="20px">
-            Pick Up
-          </Text>
-          <Box bg="white" w="100%" h="500" p={4} />
+          <HStack mb="20px">
```

Comment:
Pick up box should extend to match the end of the other box. See figma[https://www.figma.com/file/3DOUy8M8f1aWmKbpC63XMC/PNP-Wireframes?node-id=1263-10565&t=YBO0XLYmiPtG70OR-0](url)